src/UI/controllers/OverlayUIController.js

```diff
@@ -75,15 +75,19 @@ export class OverlayUIController extends UIController {
         })
         this._mutObs.observe(document.body, {childList: true, subtree: true})
 
+        const position = getPosition()
```

Comment:
The position is now stored right away, so even if it changes after the click we already know the original position and the timeout below doesn't influence it.